src/CSV.jl

```diff
@@ -175,9 +175,9 @@ Base.propertynames(row::Row{F}) where {F <: File{NamedTuple{names, T}}} where {n
 """
     CSV.File(source::Union{String, IO}; kwargs...) => CSV.File
 
-Read a csv input (a filename given as a String, or any other IO source), returning a `CSV.File` object.
```

Comment:
The convention is to use the imperative form. See https://docs.julialang.org/en/latest/manual/documentation/